src/Nethermind/Nethermind.Evm/Instruction.cs

```diff
@@ -182,9 +179,9 @@ public static class InstructionExtensions
             instruction switch
             {
                 Instruction.PREVRANDAO when !isPostMerge => "DIFFICULTY",
-                Instruction.TLOAD or Instruction.BEGINSUB => spec?.TransientStorageEnabled == true ? "TLOAD" : "BEGINSUB",
-                Instruction.TSTORE or Instruction.RETURNSUB => spec?.TransientStorageEnabled == true ? "TSTORE" : "RETURNSUB",
-                Instruction.JUMPSUB or Instruction.MCOPY => spec?.IsEip5656Enabled == true ? "MCOPY" : "JUMPSUB",
+                Instruction.TLOAD => "TLOAD",
+                Instruction.TSTORE => "TSTORE",
+                Instruction.MCOPY => "MCOPY",
```

Comment:
Can be removed, `FastEnum` will handle them well.

Reply:
Removed